lib/cocoapods/sources_manager.rb

```diff
@@ -104,7 +104,7 @@ def git(args, include_error: false)
 
     def update_git_repo(show_output = false)
       Config.instance.with_changes(:verbose => show_output) do
-        git!(%W(-C #{repo} fetch origin))
+        git!(%W(-C #{repo} fetch origin #{show_output ? "--progress" : ""}))
```

Comment:
@DanToml is that ok like this or do you have any style preference which you think I should follow?